addons/AddOns/Working/ColorSeparator/ColorSeparator.cpp

```diff
@@ -224,28 +225,30 @@ ColorSeparatorManipulatorView::ColorSeparatorManipulatorView(ColorSeparatorManip
 	target = t;
 	manipulator = manip;
 
-	BMenu *cmyk_menu = new BMenu("cmyk_menu");
+	BMenu *cmyk_menu = new BPopUpMenu("cmyk_menu");
```

Comment:
For the ColorSeparator add-on, radio buttons may be better suited than a popup menu. Also, the menu always shows "cmyk_menu" instead the selected colour.